ecocodepython.properties

```diff
@@ -24,3 +24,9 @@ defaults.mavenArtifactId=ecocode-python-plugin
 1.4.2.date=2024-01-11
 1.4.2.downloadUrl=https://github.com/green-code-initiative/ecoCode-python/releases/download/1.4.2/ecocode-python-plugin-1.4.2.jar
 1.4.2.changelogUrl=https://github.com/green-code-initiative/ecoCode-python/releases/tag/1.4.2
+
+1.4.3.description=Add support for SonarQube 10.4 "DownloadOnlyWhenRequired" feature; correction setter problem on constructor method; check Sonarqube 10.4.1 compatibility
```

Comment:
10.4?

Reply:
yes ... a previous upgrade for future 10.4 Sonarqube ... this upgrade wasn't released yet.